setup.py

```diff
@@ -18,4 +18,8 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
+    install_requires=[
+        'sqlalchemy',
+        'hdx-python-api'
+    ]
```

Comment:
I think this would satisfy Alex's request:
```suggestion
    install_requires=[
        'sqlalchemy',
   ],
   extras_require = {
        'timezone': ['hdx-python-api']
    }
```
(maybe you can think of a better name than "timezone")

However this would also require a bit of refactoring, probably moving the import of Base to its own module and performing the check proposed [here](https://humdata.slack.com/archives/C04UD1VPY2G/p1695733950349869?thread_ts=1695650246.836619&cid=C04UD1VPY2G).

Reply:
Would that refactoring needs to be done now or another sprint?

Reply:
There's no rush, so we can move it to the next sprint with the pyproject.toml migration. I'll approve this and you can merge as is!